src/invidious/views/watch.ecr

```diff
@@ -85,7 +85,15 @@ we're going to need to do it here in order to allow for translations.
                 <i class="icon ion-md-headset"></i>
             </a>
         <% end %>
+        <span class="length-watch-page"> 
+            <% if (video.length_seconds // 60) < 60 %>
+                <%= video.length_seconds // 60 %> min 
+            <% else %>
+                <%= video.length_seconds // 3600 %> hr <%= (video.length_seconds % 3600) // 60 %> min 
+            <% end %>
```

Comment:
Crystal's stdlib already provides methods to do all of these calculations for you.

```suggestion
            <% time_span_length = video.length_seconds.seconds %>

            <% if time_span_length.days > 0 %>
                <%= time_span_length.days %> d <%= time_span_length.hours %> hr <%= time_span_length.minutes %> min 
            <% elsif time_span_length.hours > 0 %>
                <%= time_span_length.hours %> hr <%= time_span_length.minutes %> min 
            <% elsif time_span_length.minutes > 0 %>
                <%= time_span_length.minutes %> min
            <% else %>
                <%= time_span_length.seconds %> seconds
            <% end %>
```